frontend/packages/operator-lifecycle-manager/src/actions/providers/subscription-provider.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { SubscriptionKind } from 'packages/operator-lifecycle-manager/src/types';
+import { useSubscriptionActions } from '../hooks/useSubscriptionActions';
+
+export const useSubscriptionActionsProvider = (resource: SubscriptionKind) => {
+  const subscriptionActions = useSubscriptionActions(resource);
+
+  const actions = React.useMemo(() => [...subscriptionActions], [subscriptionActions]);
+
+  return [actions, true];
```

Comment:
```suggestion
  return [subscriptionActions, true];
```
Until we don't provide another actions on top of the subscriptionActions for subscription resources this memoization is unnecessary, since `useSubscriptionActions` already returns memoized actions. Thinking if we want to expose the `useSubscriptionActions` directly within the `console.action/resource-provider` instead of this provider hook wrapper.